geode-assembly/src/integrationTest/resources/expected_jars.txt

```diff
@@ -44,6 +44,7 @@ jgroups
 jline
 jna
 jopt-simple
+lombok
```

Comment:
Does Lombok need to be on the runtime classpath?  If so, you'll need to make sure it gets added to geode-dependencies.jar in the `geode-assembly/build.gradle` file.